modules/api-svc/src/lib/model/OutboundRequestToPayTransferModel.js

```diff
@@ -231,12 +252,12 @@ class OutboundRequestToPayTransferModel {
      * sent because the OTP did not match.
      */
     async rejectRequestToPay() {
-        const authResponse = {
-            responseType: 'REJECTED'
+        const mojaloopResponse = {
+            transactionRequestState: 'REJECTED'
```

Comment:
It would be so nice to use enums instead of retyping out the status/states explicitly. 

Perhaps a new story is needed to address that.